### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by johnktims**

### CODE REVIEW

File: querydsl-jpa/src/main/java/com/querydsl/jpa/Conversions.java [hunk 1]
```diff
@@ -63,16 +63,16 @@ private static boolean isEntityPathAndNeedsWrapping(Expression<?> expr) {
         return false;
     }
 
-    private static <RT> FactoryExpression<RT> createEntityPathConversions(FactoryExpression<RT> factorye) {
+    private static <RT> FactoryExpression<RT> createEntityPathConversions(FactoryExpression<RT> factory) {
         List<Expression<?>> conversions = Lists.newArrayList();
-        for (Expression<?> e : factorye.getArgs()) {
+        for (Expression<?> e : factory.getArgs()) {
```
Comment (Shredder121) [inline querydsl-jpa/src/main/java/com/querydsl/jpa/Conversions.java]: Line 38 uses `factoryExpr` (I can recall it being `factorye` at first as well).
Do you think that is better to use here?

Comment (johnktims) [inline querydsl-jpa/src/main/java/com/querydsl/jpa/Conversions.java]: Indeed. I'll update it.

**Comment by timowest** [comment]
This looks good to me.

**Comment by Shredder121** [comment]
To me as well.

**Pull request merged**